### PR TITLE
Added motivation for the bundles module

### PR DESCRIPTION
**Pull request by nacx**
We should try to be very clear about the need for these bundles and why we are providing them here.

@demobox Can you have a look at the wording?

### CODE REVIEW

File: bundles/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Apache jclouds-karaf bundles
+============================
+
+This module provides convenience bundles to let jclouds run properly in OSGi environments.
+Some of the bundles contained here are not publicly available, and others are just repackaged
+to make them work well with jclouds.
```
Comment (demobox) [inline bundles/README.md]: Interesting question: where is this less likely to go out-of-date, do you think - in a README or as comments in the respective POM files?

As for the wording, I'd only just minor changes and a bit less detail (again, given the risk of this going out of date). Something like:

"This module provides bundles required to let jclouds run in OSGi environments. These bundles here may not be publicly available elsewhere, or have been repackaged to work with jclouds."

and then move detailed comments into the POM for each bundle?

**Comment by nacx** [comment]
>where is this less likely to go out-of-date, do you think - in a README or as comments in the respective POM files?

Agree with the approach of putting the details in each pom.xml and keeping the README generic.
Changes done!

**Comment by demobox** [comment]
+1 - looks good to me. Thanks, @nacx!

**Comment by demobox** [comment]
Merged to master: 522e245. Thanks, Ignasi!

**Pull request closed without merging**